src/components/Carousel/Carousel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <section class="carousel">
+    <div class="carousel__container">
+      <button
+        id="left-btn"
+        class="carousel-control carousel-control--left"
+        @click="handlePrevious"
+      >
+        <img
+          src="../../assets/icons/left.svg"
```

Comment:
You could try out if you are curious: it might be possible to import this in js code, define as component and use it here inline 🤔